docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+However, this may not always be the case and it should be considered when
+relevant.
+
+Prevalence data can be used to **initialize cause model states** and
+represents the **probability that a simulant will begin the simulation in a
+given state.**
+
+  For example, the probability that a simulant in a model of obesity in the
+  United States beginning in 2016 will begin the simulation as obese is 0.4 or
+  40%.
+
+Birth Prevalence
 ++++++++++++++++
 
-Remission
+Birth prevalence is defined as the **proportion of live births in a given
+population that possess a given condition or trait at birth.**
+
+  For example, the birth prevalence for cleft lip in the United States in 2006
+  was 10.6 per 10,000 live births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model
+states** and represent the **probability that a simulant who is born during
+the simulation will be born into a given neonatal cause model state.**
+
+  For example, the probability that a simulant born during a simulation of
+  cleft lip in the United States in 2006 is 0.00106, or 0.106%.
+
+Incidence
 +++++++++
 
-Prevalence
+Incidence rates are defined as the **number of new cases of a condition that
+occur per person-year of the at-risk population (individuals without
+condition).** Specifically, the at-risk population can be represented as
+`1 - condition prevalence`.
+
+  For example, the incidence of multiple sclerosis (MS) in the United States
+  is 2.8 per 100,000 person-years of the at-risk population.
+
+  This suggests that if we followed 100,000 individuals without MS for 1
+  year each (100,000 people * 1 year = 100,000 person-years), we would expect
+  2.8 of these individuals to develop MS within this timeframe.
+
+  Alternatively, if we followed 50,000 individuals without MS for 2 years each
+  (50,000 people * 2 years = 100,000 person-years), we also would expect 2.8
+  of these individuals to develop MS within this timeframe.
+
+Incidence can be used to **estimate cause model transition rates** and can
+represent the **probability that a simulant will transition from a susceptible
+state to an infected state within a given timestep.**
+
+  For example, with a timestep of one year and using incidence as the
+  transition rate data source, the probability that a simulant will transition
+  from a susceptible (without MS) cause model state to an infected (with MS)
+  cause model state is 2.8*10^(-5).
+
+.. _above:
+
+**A Few Considerations for Incidence Data Sources:**
+
+As mentioned above, the denominator for incidence is person-years of the
+*at-risk* population, or the population *without* condition (``1 - condition
+prevalence``). However, in certain scenarios, this may not always be the
+case.
+
+  In situations when the general population is represented in the denominator
+  rather than the at risk population...
+
+
+    If the prevalence of a condition is *small*, ``1 - prevalence`` ~ ``1``.
```

Comment:
This is not necessarily true.  Consider a low-prevalence, unrecoverable, morbidity only condition (e.g. amputee or untreatable blindness).  Because using the whole population in the denominator **biases** our results to be low, you'll see significant divergence in the disease prevalence in sufficiently long simulations.  

The point being, it's an okay approximation if you're just trying to do some data comparisons, but you should take care using biased approximators for simulation as it can have dramatic consequences.

Reply:
updated this to just say that it will be biased, especially when prevalence is large